### ISSUE TITLE
Cell execution is faster in the background

### ISSUE THREAD

**Issue by Aggrathon** [microsoft/vscode-jupyter#11614]
### Applies To

- [X] Notebooks (.ipynb files)
- [ ] Interactive Window and\/or Cell Scripts (.py files with \#%% markers)

### What happened?

**Expected:** Notebooks execute as fast (or faster) in the foreground as when looking at a another tab.  
**Experienced:** Switching to another tab can speed up the execution of Python cells.

**Steps to reproduce:**  
1. Create a new notebook.
2. Create a cell with a heavy computation, e.g.:
```python
for i in range(10_000):
    [i**3.5 for i in range(10_000)]
```
3. Run it (took 15.1s on this laptop).
4. Run it again but immediately switch to another tab (e.g. by CTRL+N).
5. Wait for a bit and then check the time (took 9.5s on this laptop).

In this case I get only a ~30% time saving by switching to another tab, but anecdotally I've seen more than 50% (for some heavy seaborn plots). 

### VS Code Version

Version: 1.72.0 Commit: 64bbfbf67ada9953918d72e1df2f4d8e537d340e Date: 2022-10-04T23:20:47.539Z Electron: 19.0.17 Chromium: 102.0.5005.167 Node.js: 16.14.2 V8: 10.2.154.15-electron.0 OS: Linux x64 5.15.0-48-generic Sandboxed: No

### Jupyter Extension Version

v2022.9.1002791758

### Coding Language and Runtime Version

Python v3.9.7

### Language Extension Version (if applicable)

v2022.16.0

### Running Jupyter locally or remotely?

Local

**Comment by roblourens** [comment]
For me, it takes about 10s whether in the foreground or background, pretty consistently after trying several times. With this code, I don't see how we could be affecting it because that time is just python running code. If you see a difference in something that is renderer heavy, esp that produces streaming text, that could actually make sense because when a tab is in the background, we might save time by not actually rendering the intermediate result. Are you sure you see a large difference reliably with that code above?

**Comment by Aggrathon** [comment]
I tried the same on another computer, and didn't see any difference. So back on the original laptop I monitored active processes and found that two VS Code processes consumed CPU while rendering the "cell is executing" animation (which obviously wasn't showing when I switched to another tab). Curiously, disabling hardware acceleration in VS Code almost removed the overhead completely (I'm not sure why yet).

In summary: showing the "executing" animation had a much greater CPU overhead than I thought possible (on this not very impressive laptop). And, due to the improvement when disabling hardware acceleration, I think the issue is not with this extension.
